ql/src/test/org/apache/hadoop/hive/ql/exec/vector/expressions/TestMurmurHashExpression.java

```diff
@@ -241,4 +241,143 @@ public void testMurmurHashWithNullsInt() throws HiveException {
           ((LongColumnVector) vrb.cols[2]).vector[i]);
     }
   }
+
+  @Test
```

Comment:
There seems to be a bit of duplication in the tests. Do we have anything to factor out?

Reply:
I refactored `ColumnVector` creation part to reduce code redundancy and stop using random vector.